### DISCUSSION TITLE
Tutorial video

### DISCUSSION THREAD

**Discussion by janoJJJ** [sunecz/Media-Downloader#5] [General]
Videl som ako appka funguje na Windowse a veľmi sa mi páčila. Nepríjemným prekvapením je extrémna zložitosť  nainštalovať ju na Linux. Návod sa ani nedá nazvať návodom ale skôr dvojdňovým trápením a frustrovanou rezignáciou dosiahnuť nemožné. Nemohol by niekto urobiť podrobný video návod napr. na Youtube. Z návodu na Githube som 90 percent ničomu nerozumel a potrebujem slepo opakovať úkony podľa video návodu. Dakujem

**Comment by sunecz** [comment]
Zdravím, návod na GitHubu je pro ty, kteří si chtějí program sestavit sami. Jinak ho lze normálně stáhnout přes oficiální stránky: https://projects.suneweb.net/media-downloader/ (vyberte "Linux (64-bit)") Stačí rozbalit archív a spustit soubor Media Downloader, buď přes GUI dvojklikem, nebo v terminálu příkazem `./Media\ Downloader`. Momentálně je doporučeno použít předběžnou verzi, návod je také na stránkách, ale pro jistotu i zde: "V horní liště klikněte na Program > Nastavení, záložka Obecné, tam přepěnete Používat předběžné verze na hodnotu Do další hlavní verze. Dejte Uložit a následně restartujte program. Předběžná verze se pak automaticky stáhne."

**Reply by janoJJJ** [reply]
Bože ja som debil. Keby som neprekombinoval vyhľadávanie na nete zamerané na návod ale skôr jednoducho na stánku pre stiahnutie appky nikdy by som dva dni nepresedel v miernej depke nad laptopom. Vdaka. Problem vyriešený.

**Reply by sunecz** [reply]
Tak snad už mít depku nebudete a ať je Vám program užitečný! :)
